run-tests.sh

```diff
@@ -2,10 +2,26 @@
 INPUT_DATA=$1
 IMAGETAG=$2
 
-# Test if the CLI actually works (PNG)
-for i in $(ls $INPUT_DATA/*.mmd); do docker run -v $(pwd):/data $IMAGETAG -i /data/$i -o /data/$i.png -w 800; done
-for i in $(ls $INPUT_DATA/*.mmd); do cat $i | docker run -i -v $(pwd):/data $IMAGETAG -o /data/$i-stdin.png -w 800; done
+# Test if the CLI actually works (PNG, PDF and SVG)
+# If there is a corresponding .css for a test, it will be used as an additional parameter: -C <css-file>
+
+for test in $(ls $INPUT_DATA/*.mmd); do
+  STYLE_PARAMS=""
+
+  STYLE=$INPUT_DATA/`basename $test .mmd`.css
+  if [ -f $STYLE ] ; then
+    STYLE_PARAMS="-C /data/$STYLE"
+  fi
+
+  for fmt in png pdf svg; do
+    EXTRA_PARAMS=""
+
+    if [ "$fmt" = "png" ] ; then
+      EXTRA_PARAMS="-w 800"
+    fi
+    docker run -v $(pwd):/data $IMAGETAG -i /data/$test $STYLE_PARAMS -o /data/$test.$fmt $EXTRA_PARAMS
```

Comment:
```suggestion
    docker run -v $(pwd):/data $IMAGETAG -i /data/$test $STYLE_PARAMS -o /data/$test-$fmt.$fmt $EXTRA_PARAMS
```
I would suggest to make output filenames more unique. Otherwise, the svg files converted to png will overwrite png files with the same name.